src/pipeline.runner.ts

```diff
@@ -159,7 +167,8 @@ export class PipelineRunner {
         let releaseStartMetadata: ReleaseInterfaces.ReleaseStartMetadata = <ReleaseInterfaces.ReleaseStartMetadata>{
             definitionId: releaseDefinition.id,
             reason: ReleaseInterfaces.ReleaseReason.ContinuousIntegration,
-            artifacts: artifacts
+            artifacts: artifacts,
+            variables: JSON.parse(this.taskParameters.azurePipelineVariables)

```

Comment:
Should this be using the `variables` variable from above instead of reparsing the variable string here?

```suggestion
            variables: variables
```